src/Common/src/Internal/Cryptography/UniversalCryptoDecryptor.cs

```diff
@@ -97,7 +97,18 @@ protected sealed override byte[] UncheckedTransformFinalBlock(byte[] inputBuffer
             byte[] outputData;
             if (ciphertext.Length > 0)
             {
-                outputData = DepadBlock(decryptedBytes, 0, decryptedBytes.Length);
+                unsafe
+                {
+                    fixed (byte* decryptedBytesPtr = decryptedBytes)
```

Comment:
I'd put a comment about GC compaction, so it doesn't get accidentally removed in the future

Reply:
If it was somewhere other than crypto code, I'd agree.  But there's already plenty of usage of "unused byte*" in crypto code, I think that diminishing returns has set in on commenting it.